demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1445,9 +1470,43 @@ export class DemoMeetingApp implements
   async populateAllDeviceLists(): Promise<void> {
     await this.populateAudioInputList();
     await this.populateVideoInputList();
+    await this.populateVideoFilterInputList();
     await this.populateAudioOutputList();
   }
 
+
+  private async populateVideoFilterInputList(): Promise<void> {
+    const genericName = 'Filter';
+    let filters: VideoFilterName[] = ['None'];
+
+    if (this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() && this.enableUnifiedPlanForChromiumBasedBrowsers) {
+      filters = filters.concat(VIDEO_FILTERS);
+      if (this.defaultBrowserBehaviour.supportsVideoProcessingBasedOnTensorflowModel()) {
```

Comment:
This is a great place to lazy-load dependencies.

Reply:
I found something like this 

```
function loadScript(url: string, sri: string, crossOrigin: string): void {
  var s = document.createElement('script');
  s.type = 'text/javascript';
  s.async = true;
  s.src = url;
  s.integrity = sri;
  s.crossOrigin = crossOrigin;
  var x = document.getElementsByTagName('head')[0];
  x.appendChild(s);
}

```

so we could lazy load

Reply:
Yeah, something like that, but with more descriptive names and no use of `var`.

Reply:
sounds good. I will use such lazy-loading script when putting segmentation filter demo back in